<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextEditor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resources/images/text_editor_icon.ico</normaloff>resources/images/text_editor_icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="statusTip">
       <string>Enter the text</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="action_Preview"/>
    <addaction name="action_Print"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;About</string>
    </property>
    <addaction name="action_About_program"/>
    <addaction name="action_About_Qt"/>
   </widget>
   <widget class="QMenu" name="Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="action_Settings"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Zoom_in"/>
    <addaction name="action_Zoom_out"/>
    <addaction name="action_Default_zoom"/>
   </widget>
   <widget class="QMenu" name="menu_Format">
    <property name="title">
     <string>&amp;Format</string>
    </property>
    <widget class="QMenu" name="menu_Color">
     <property name="title">
      <string>&amp;Color</string>
     </property>
     <addaction name="action_Text"/>
     <addaction name="action_Background"/>
    </widget>
    <addaction name="action_Font"/>
    <addaction name="menu_Color"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="separator"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="separator"/>
    <addaction name="action_Select_all"/>
    <addaction name="action_Date_and_time"/>
    <addaction name="separator"/>
    <addaction name="action_Go_to_the"/>
    <addaction name="action_Find"/>
    <addaction name="action_Replace"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Format"/>
   <addaction name="menu_View"/>
   <addaction name="Settings"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_Undo"/>
   <addaction name="action_Redo"/>
   <addaction name="separator"/>
   <addaction name="action_Cut"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Paste"/>
   <addaction name="separator"/>
   <addaction name="action_Settings"/>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset>
     <normaloff>resources/images/new.ico</normaloff>resources/images/new.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="statusTip">
    <string>Create a new file</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset>
     <normaloff>resources/images/open.ico</normaloff>resources/images/open.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="statusTip">
    <string>Open a file from the selected directory</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset>
     <normaloff>resources/images/save.ico</normaloff>resources/images/save.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="statusTip">
    <string>Save file in selected directory</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset>
     <normaloff>resources/images/exit.ico</normaloff>resources/images/exit.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit from application</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resources/images/undo.ico</normaloff>resources/images/undo.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="statusTip">
    <string>Undo the last action</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resources/images/redo.ico</normaloff>resources/images/redo.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="statusTip">
    <string>Restore a canceled action</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resources/images/cut.ico</normaloff>resources/images/cut.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="statusTip">
    <string>Delete the selected text</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset>
     <normaloff>resources/images/paste.ico</normaloff>resources/images/paste.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="statusTip">
    <string>Paste the text from the clipboard</string>
   </property>
  </action>
  <action name="action_Select_all">
   <property name="icon">
    <iconset>
     <normaloff>resources/images/select_all.ico</normaloff>resources/images/select_all.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Select all</string>
   </property>
   <property name="statusTip">
    <string>Select all the contents of the file</string>
   </property>
  </action>
  <action name="action_About_program">
   <property name="icon">
    <iconset>
     <normaloff>resources/images/about_program.ico</normaloff>resources/images/about_program.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;About program</string>
   </property>
   <property name="statusTip">
    <string>View the program version</string>
   </property>
  </action>
  <action name="action_About_Qt">
   <property name="icon">
    <iconset>
     <normaloff>resources/images/about_qt.ico</normaloff>resources/images/about_qt.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;About Qt</string>
   </property>
   <property name="statusTip">
    <string>Information window of Qt</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resources/images/copy.ico</normaloff>resources/images/copy.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="statusTip">
    <string>Copy the selected text fragment</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="icon">
    <iconset>
     <normaloff>resources/images/settings.ico</normaloff>resources/images/settings.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
   <property name="statusTip">
    <string>Ð¡ustomize the view of the status bar and toolbar</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Date_and_time">
   <property name="icon">
    <iconset>
     <normaloff>resources/images/date_time.ico</normaloff>resources/images/date_time.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Date and time</string>
   </property>
   <property name="statusTip">
    <string>Show current date and time</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_Go_to_the">
   <property name="icon">
    <iconset>
     <normaloff>resources/images/go_to.ico</normaloff>resources/images/go_to.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Go to . . .</string>
   </property>
   <property name="statusTip">
    <string>Go to the line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_Font">
   <property name="text">
    <string>&amp;Font</string>
   </property>
   <property name="statusTip">
    <string>Choose a font</string>
   </property>
  </action>
  <action name="action_Text">
   <property name="text">
    <string>&amp;Text</string>
   </property>
   <property name="statusTip">
    <string>Choose color of text</string>
   </property>
  </action>
  <action name="action_Background">
   <property name="text">
    <string>&amp;Background</string>
   </property>
   <property name="statusTip">
    <string>Choose background color</string>
   </property>
  </action>
  <action name="action_Zoom_in">
   <property name="text">
    <string>&amp;Zoom in</string>
   </property>
   <property name="statusTip">
    <string>Zoom in on the text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="action_Zoom_out">
   <property name="text">
    <string>&amp;Zoom out</string>
   </property>
   <property name="statusTip">
    <string>Zoom out on the text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="action_Default_zoom">
   <property name="text">
    <string>&amp;Default zoom</string>
   </property>
   <property name="statusTip">
    <string>Make the default text zoom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="action_Word_wrap">
   <property name="text">
    <string>&amp;Word wrap</string>
   </property>
  </action>
  <action name="action_Label">
   <property name="text">
    <string>&amp;Label</string>
   </property>
  </action>
  <action name="action_Preview">
   <property name="text">
    <string>&amp;Preview</string>
   </property>
   <property name="statusTip">
    <string>Preview the current text file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="statusTip">
    <string>Print the current text file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Left">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Center">
   <property name="text">
    <string>&amp;Center</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Right">
   <property name="text">
    <string>&amp;Right</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Justify">
   <property name="text">
    <string>&amp;Justify</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Find">
   <property name="text">
    <string>&amp;Find</string>
   </property>
   <property name="statusTip">
    <string>Find text fragment in the file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_Replace">
   <property name="text">
    <string>&amp;Replace</string>
   </property>
   <property name="statusTip">
    <string>Find text fragment and replace it with another one</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>textEdit</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>action_Cut</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEdit</sender>
   <signal>undoAvailable(bool)</signal>
   <receiver>action_Undo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEdit</sender>
   <signal>redoAvailable(bool)</signal>
   <receiver>action_Redo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Paste</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Undo</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Select_all</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Redo</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEdit</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>action_Copy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
